packages/now-python/test/fixtures/00-request-path/vercel.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": 2,
+  "builds": [{ "src": "api/*.py", "use": "@vercel/python" }],
+  "probes": [
+    {
+      "path": "/api/python.py?query=hello%20%40world",
+      "mustContain": "query=hello%20%40world"
+    },
+    { "path": "/api/python.py?hello=%2F", "mustContain": "?hello=%2F" },
```

Comment:
What about `/api/python.py?hello=/` ? What happens then?

Reply:
Diverging behavior, stemming from the proxy:

* Proxy will URL-encode the `/` if it's after the `?` before passing it on, so the server function would receive `/api/python.py?hello=%2F`. I can't think of anything wrong with doing this, but at the same time I feel like proxy should do as little modifications to the request as possible, because the handling of the original URL may be opinionated (redirecting double slashes is another example where I think proxy gets in the way and prevents certain use cases).
* `fetch()` doesn't automatically URL-encode the `/`, so the Python server receives the original `/api/python.py?hello=/`